### DISCUSSION TITLE
war within broken

### DISCUSSION THREAD

**Discussion by rhyenvi** [wardz/DRList-1.0#25] [General]
hey, wardz! what happened with war within? GetSpellInfo is broken? so many addons rely on you! thank you, sir!

**Comment by wardz** [comment]
Update the lib to v1.5.2+ and it should work

I'm on vacation for another week so i wont be able to add any of the new DR spells yet. (if any)

**Comment by rhyenvi** [comment]
thanks a lot for the reply. will try that
